src/pages/ProductDetail/ProductTab.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import './ProductTab.scss';
+
+const ProductTab = () => {
+  const data = [
+    {
+      id: 0,
+      title: '상품정보',
+      src: './images/kunwooSample/productInfo.png',
+    },
+    {
+      id: 1,
+      title: '교환/반품',
+      src: './images/kunwooSample/change.png',
+    },
+  ];
+  const [index, setIndex] = useState(0);
+  return (
+    <section className="tab">
+      <div className="tab-menu">
+        {data.map(item => (
+          <div className="tabtab">
+            <button
+              key={item.id}
+              className={index === item.id ? 'btn' : 'btn-white'}
+              onClick={() => setIndex(item.id)}
+            >
+              {item.title}
+            </button>
+          </div>
+        ))}
+      </div>
+      {data
+        .filter(item => index === item.id)
+        .map(item => (
+          <div className="tab-content">
+            <img src={item.src} />
+          </div>
+        ))}
```

Comment:
현재 탭의 이미지를 보여주는 거라면, map 메서드를 또 체이닝할 필요 없이 다음처럼만 사용할 수 있을 것 같습니다.
```suggestion
      <div className="tab-content">
        <img src={data.find(({ id }) => currTabId === id).src} />
      </div>
```

Reply:
감사합니다

Reply:
변경 완료하였습니다